ClosedXML/Excel/CalcEngine/XLCalcEngine.cs

```diff
@@ -90,7 +90,7 @@ public IEnumerator GetEnumerator()
         // ** implementation
         private object GetValue(IXLCell cell)
         {
-            if (_evaluating)
+            if (_evaluating || (cell is XLCell && (cell as XLCell).IsEvaluating))
```

Comment:
I think you can safely assume that `cell is XLCell` will always be `true`. If not, basically everything in ClosedXML will break.